### ISSUE TITLE
Rails console requiring readline

### ISSUE THREAD

**Issue by rhuanbarreto** [rails/rails#35439]
### Steps to reproduce

- Install an Rails Application.
- run `rails c` command

### Expected behavior

The rails console to be up and running

### Actual behavior

It gives an error:
```
Running via Spring preloader in process 38353
Traceback (most recent call last):
        50: from -e:1:in `<main>'
        49: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        48: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        47: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `load'
        46: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
        45: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `block in load'
        44: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:50:in `load'
        43: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:50:in `load'
        42: from /Users/rhuan/code/odoo/bin/rails:9:in `<main>'
        41: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
        40: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
        39: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
        38: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
        37: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
        36: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
        35: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
        34: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
        33: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/commands.rb:18:in `<main>'
        32: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command.rb:44:in `invoke'
        31: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command.rb:70:in `find_by_namespace'
        30: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:79:in `lookup'
        29: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:79:in `each'
        28: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:80:in `block in lookup'
        27: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:80:in `each'
        26: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:84:in `block (2 levels) in lookup'
        25: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
        24: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
        23: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
        22: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
        21: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
        20: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
        19: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
        18: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
        17: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/commands/console/console_command.rb:4:in `<main>'
        16: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
        15: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
        14: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
        13: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
        12: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
        11: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
        10: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
         9: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
         8: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/irb/completion.rb:10:in `<main>'
         7: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
         6: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
         5: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
         4: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
         3: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
         2: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
         1: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require': dlopen(/Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle, 9): Library not loaded: /usr/local/opt/readline/lib/libreadline.7.dylib (LoadError)
  Referenced from: /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle
  Reason: image not found - /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle
```

### System configuration
**Rails version**: 5.2.2
**Ruby version**: ruby 2.6.0p0 (2018-12-25 revision 66547) [x86_64-darwin18]
**OS**: MacOs Mojave

### Temporary Measure

Creating a symlink from the old version solves the problem. As described in this [StackOverflow question](https://stackoverflow.com/questions/40671803/library-not-loaded-usr-local-opt-readline-lib-libreadline-6-dylib-loaderror)

`ln -s /usr/local/opt/readline/lib/libreadline.8.0.dylib /usr/local/opt/readline/lib/libreadline.7.dylib`

**Comment by jackozi** [comment]
Reinstalling your ruby version also fixes this issue `rbenv install 2.6.0`

**Comment by robin850** [comment]
Hi @rhuanbarreto,

Thanks for reporting but this rather looks like an issue with your setup or Rbenv.

**Comment by bradical** [comment]
Hi I just ran into this too and while this is a good suggestion:

> Reinstalling your ruby version also fixes this issue rbenv install 2.6.0

I'm not in a position to upgrade ruby at the moment. Would just reinstalling my current ruby version: `2.4.3` work as well? Anyone know what the cause of this is?

**Comment by stager94** [comment]
> ### Steps to reproduce
> * Install an Rails Application.
> * run `rails c` command
> 
> ### Expected behavior
> The rails console to be up and running
> 
> ### Actual behavior
> It gives an error:
> 
> ```
> Running via Spring preloader in process 38353
> Traceback (most recent call last):
>         50: from -e:1:in `<main>'
>         49: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
>         48: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
>         47: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `load'
>         46: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
>         45: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `block in load'
>         44: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:50:in `load'
>         43: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:50:in `load'
>         42: from /Users/rhuan/code/odoo/bin/rails:9:in `<main>'
>         41: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
>         40: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
>         39: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
>         38: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
>         37: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
>         36: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
>         35: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
>         34: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
>         33: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/commands.rb:18:in `<main>'
>         32: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command.rb:44:in `invoke'
>         31: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command.rb:70:in `find_by_namespace'
>         30: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:79:in `lookup'
>         29: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:79:in `each'
>         28: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:80:in `block in lookup'
>         27: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:80:in `each'
>         26: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/command/behavior.rb:84:in `block (2 levels) in lookup'
>         25: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
>         24: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
>         23: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
>         22: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
>         21: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
>         20: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
>         19: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
>         18: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
>         17: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/commands/console/console_command.rb:4:in `<main>'
>         16: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
>         15: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
>         14: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
>         13: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
>         12: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
>         11: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
>         10: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
>          9: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
>          8: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/irb/completion.rb:10:in `<main>'
>          7: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
>          6: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
>          5: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
>          4: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
>          3: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
>          2: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
>          1: from /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
> /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require': dlopen(/Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle, 9): Library not loaded: /usr/local/opt/readline/lib/libreadline.7.dylib (LoadError)
>   Referenced from: /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle
>   Reason: image not found - /Users/rhuan/.rbenv/versions/2.6.0/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle
> ```
> 
> ### System configuration
> **Rails version**: 5.2.2
> **Ruby version**: ruby 2.6.0p0 (2018-12-25 revision 66547) [x86_64-darwin18]
> **OS**: MacOs Mojave
> 
> ### Temporary Measure
> Creating a symlink from the old version solves the problem. As described in this [StackOverflow question](https://stackoverflow.com/questions/40671803/library-not-loaded-usr-local-opt-readline-lib-libreadline-6-dylib-loaderror)
> 
> `ln -s /usr/local/opt/readline/lib/libreadline.8.0.dylib /usr/local/opt/readline/lib/libreadline.7.dylib`
